openshift/topics/get_started.adoc

```diff
@@ -23,10 +23,18 @@ See the link:https://access.redhat.com/RegistryAuthentication[Red Hat Container
 To update to the latest {project_openshift_product_name} images, run the following commands:
 
 . On your master host(s), ensure that you are logged in as a cluster administrator or a user with project administrator access to the global `openshift` project.
+
+.. For OpenShift 3.x, use this command:
```

Comment:
@andymunro I would suggest to change these steps as described in the [corresponding JIRA comment](https://issues.redhat.com/browse/KEYCLOAK-13280?focusedCommentId=14007347&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14007347)

The reason is on OCP v3, the user logged in remotely via `oc` CLI needs to be granted the `cluster-admin` role first (in order to be able to edit / update the image streams / templates) definitions in the `openshift` namespace / project. So I would keep the current version.

But for OCP v4 the suggestion is correct (`kubeadmin` user has this privilege / role by default). So this truly is the preferred method of updating the image streams / templates (since SSH access to underlying RHCOS nodes in the case of OCP v4 cluster is discouraged).